### ISSUE TITLE
Up/Down buttons on the Search widget don't navigate through search results

### ISSUE THREAD

**Issue by yurivict** [xournalpp/xournalpp#2652]
**Affects versions :**
 - OS: FreeBSD 12.2
 - (Linux only) Desktop environment: Xfce4
 - Which version of libgtk do you use: 3.24.24
 - Version of Xournal++: 1.0.20
 - Installation method: package

**Describe the bug**
Large single-page PDF is loaded. The PDF was produced by the ```dot``` utility from ```GraphViz```.
Ctrl-F brings focus to the search widget.
Typing some text shows the prompt: "Text 8 times found on this page" (please fix the grammar too!)
But pressing Up/Down buttons doesn't navigate between the text occurrences on the page.

**Expected behavior**
Navigation should be working from the Search widget.

**Comment by rolandlo** [comment]
Not sure it is a bug. It could be intended behaviour. Just markup all occurences on one page. Pressing Up/Down will relocate you to the next/previous page, where the search string appears.

**Comment by yurivict** [comment]
> Pressing Up/Down will relocate you to the next/previous page, where the search string appears.

If it can relocate to the page but not to the other location on the page - this is quite a confusing behavior.

Web browsers, for example,  have a similar "search" feature and it simply relocates the view to the location.

**Comment by Technius** [comment]
I agree with @yurivict, I had to use the search feature yesterday and it is definitely not intuitive. It only counts text on the same page, and it jumps around strangely.
